app/webpacker/components/EditSchedule/EditVenues/VenuePanel.js

```diff
@@ -121,7 +121,8 @@ function VenuePanel({
     isError: timeZonesError,
   } = useQuery({
     queryFn: fetchSuggestedTimeZones,
-    queryKey: ['suggested-tz', venue.id, venue.latitudeMicrodegrees, venue.longitudeMicrodegrees],
+    queryKey: ['suggested-tz', venue.latitudeMicrodegrees, venue.longitudeMicrodegrees],
+    enabled: venue.latitudeMicrodegrees && venue.longitudeMicrodegrees,
```

Comment:
Unlikely scenario in practice, but if one of the coordinates is 0 then this will be disabled, I think.